google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/stub/metrics/BuiltinMetricsTracer.java

```diff
@@ -355,6 +378,15 @@ private void recordAttemptCompletion(@Nullable Throwable status) {
     attemptLatenciesHistogram.record(
         convertToMs(attemptTimer.elapsed(TimeUnit.NANOSECONDS)), attributes);
 
+    if (attemptCount == 1) {
```

Comment:
And maybe change this to <= 1, just in case for some reason the attempt counter is not incremented.